README.md

```diff
@@ -219,7 +224,7 @@ new AppsyncResolver(() => {
 });
 ```
 
-### `AppsyncResolver` construct for defining Appsync Resolver with TypeScript syntax
+#### `AppsyncResolver` construct for defining Appsync Resolver with TypeScript syntax
```

Comment:
Why did this change?

Reply:
I think I bumped all of those headers down a level.
![image](https://user-images.githubusercontent.com/289213/161288308-cf1b23fb-0c13-4a1e-be7f-b045ecec6b4c.png)

It was

```
## Usage
### `AppsyncResolver` construct for defining Appsync Resolver with TypeScript syntax
```

and I made it 

```
## Usage
### App Sync
#### `AppsyncResolver` construct for defining Appsync Resolver with TypeScript syntax
### Event Bridge
#### EB Stuff
```